spec.html

```diff
@@ -23072,7 +23284,380 @@ <h1>Example Cyclic Module Record Graphs</h1>
+
+          <emu-table id="table-module-graph-cycle-async-fields-1" caption="Module fields after the initial Evaluate() call">
+            <table>
+              <thead>
+                <tr>
+                  <th>Module</th>
+                  <th>[[DFSIndex]]</th>
+                  <th>[[DFSAncestorIndex]]</th>
+                  <th>[[Status]]</th>
+                  <th>[[AsyncEvaluation]]</th>
+                  <th>[[AsyncParentModules]]</th>
+                  <th>[[PendingAsyncDependencies]]</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <th>_A_</th>
+                  <td>0</td>
+                  <td>0</td>
+                  <td>~evaluating-async~</td>
+                  <td>*true*</td>
+                  <td>&laquo; &raquo;</td>
+                  <td>2 (_B_ and _C_)</td>
+                </tr>
+                <tr>
+                  <th>_B_</th>
+                  <td>1</td>
+                  <td>0</td>
+                  <td>~evaluating-async~</td>
+                  <td>*true*</td>
+                  <td>&laquo; _A_ &raquo;</td>
+                  <td>1 (_D_)</td>
+                </tr>
+                <tr>
+                  <th>_C_</th>
+                  <td>2</td>
+                  <td>0</td>
+                  <td>~evaluating-async~</td>
+                  <td>*true*</td>
+                  <td>&laquo; _A_ &raquo;</td>
+                  <td>2 (_D_ and _E_)</td>
+                </tr>
+                <tr>
+                  <th>_D_</th>
+                  <td>3</td>
+                  <td>0</td>
+                  <td>~evaluating-async~</td>
+                  <td>*true*</td>
+                  <td>&laquo; _B_, _C_ &raquo;</td>
+                  <td>0</td>
+                </tr>
+                <tr>
+                  <th>_E_</th>
+                  <td>4</td>
+                  <td>4</td>
+                  <td>~evaluating-async~</td>
```

Comment:
The end of the previous paragraph says this table is the state at that point in time, which should already have E evaluated, no?